README.md

```diff
@@ -1,131 +1,231 @@
-
-```bash
-ae free
-ae free cell
-ae free pod
-```
-
-Free resources, and destroy any prerequisites that have been started. Will fail if resources cannot be freed or do not exist.
-
-```bash
-ae start
-ae start executable, exe
-ae start container # Note this has an alias: 'ae oci start'
-```
-
-Run a resource immediately.
-
-```bash
-ae stop
-ae stop executable, exe
-ae stop container # Note this has an alias: 'ae oci kill'
-```
-
-Stop a resource immediately.
-
-#### Logs
-
-```bash
-ae logs <options>
-```
-
-#### Discovery
-
-```bash
-ae discover <cidr <cidr> | ip <ip>>
-```
-
-Scans a cluster of Aurae nodes and returns information about them, including the version running.
-
-#### Health
-
-```bash
-ae check <cidr <cidr> | ip <ip>> <service,...>
-```
-
-Scans a cluster of Aurae nodes and returns the serving status of the given list of services.
-
-
-## Guiding Principles
-
-### Less is more.
+&nbsp;
+
+<!-- ABOUT THE PROJECT -->
+<h2 id="about-the-project">About The Project</h2>
 
+`ae` is a UNIX inspired CLI client for **[Aurae](https://github.com/aurae-runtime/aurae)**, written in Go. However, in order to understand what `ae` should and can do, we must first understand `aer`.
+
+### **What is `aer`?**
+
+The intention is to build an automatically generated CLI tool called `aer`, which is based on the Rust client that has the identical scope of a single _[auraed](https://github.com/aurae-runtime/aurae/tree/main/auraed)_ node.
+
+The tool will be aimed at "POWER-USERS" and exists as a rapid way to develop and debug against APIs that change frequently. For example, an [auraed](https://github.com/aurae-runtime/aurae/tree/main/auraed) developer can make a change to an existing API and test it locally against a single daemon.
+
+
+### **What is `ae`?**
+
+We want to create and maintain a CLI tool, `ae` that is based on the Go client and has a broader view than `aer`.
+
+The purpose will be to use `ae` for clusters of [Aurae](https://github.com/aurae-runtime/aurae) nodes and will probably work similar to `aer`, but not completely!
+
+The `ae` CLI tool will work for a group of nodes and will probably contain more pragmatic functions that are more important for enterprise operators.
+
+### **Who is `ae` for?**
+
+The `ae` utility should be familiar to every cloud operator.
+
+_Typical tasks_ such as:
+
+* "Rolling out NGINX to production"
+* "Counting the numbers of threads of a process"
+* "Changing the current databse connection count for a service"
+* "Swinging traffic from one to another"
+
+should be possible with `ae`.
+
+There are more "practical" and "impact-oriented" tasks, and these probably need extra functionality, which they add to the lightweight `aer` tool.
+
+<!-- QUICKSTART -->
+<h2 id="about-the-project">Quickstart</h2>
+
+> This section is reserved for future installation instructions as well as an example of integration.
+
+<!-- USAGE -->
+<h2 id="about-the-project">Usage</h2>
```

Comment:
i don't think we should list this in the README.  i'm open to being corrected, but this is another place where we need to keep things up-to-date while the source of truth is `ae --help`.

Reply:
I am actually not well-read in this subject and have only been inspired by many other CLI tools, including well-known ones. Others would have to let you know or you would have to make a decision on your own...

Reply:
we can leave it for now